scripts/install/install-mac.sh

```diff
@@ -27,5 +27,5 @@ $PORTER_HOME/porter mixin install gcloud --version $PKG_PERMALINK
 $PORTER_HOME/porter plugin install azure --version $PKG_PERMALINK
 
 echo "Installation complete."
-echo "Add porter to your path by running:"
+echo "Add porter to your path by adding the following line to your ~/.bash_profile:"
```

Comment:
I forgot that the latest version of MacOS (Catalina) changed the default shell from Bash to ZSH. Their profile file is `~/.zprofile`. Let's change this a bit to account for the user having either:

```suggestion
echo "Add porter to your path by adding the following line to your ~/.bash_profile or ~/.zprofile:"
```